instructure/canvas-ios
# Student - Automation - E2E test for NewQuiz

refs: MBL-17340
affects: None
release note: None
test plan: Tests to pass

### testNewQuiz

- Sets feature flag to enable NewQuiz
- Creates a student user, a course, and a new type of quiz and a quiz item with DataSeeder
- Gets the user logged in
- Navigates to Quizzes
- Check if the newly created quiz is displayed
- Tap on quiz and check detail screen
- Tap on Launch External Tool button and check the URL of the opened page
